opentsdb/getmetrics_opentsdb.py

```diff
@@ -24,144 +29,224 @@ def getParameters():
                       action="store", dest="chunkSize", help="Metrics per chunk")
     (options, args) = parser.parse_args()
 
-    parameters = {}
+    params = {}
     if options.homepath is None:
-        parameters['homepath'] = os.getcwd()
+        params['homepath'] = os.getcwd()
     else:
-        parameters['homepath'] = options.homepath
-    if options.serverUrl == None:
-        parameters['serverUrl'] = 'https://app.insightfinder.com'
+        params['homepath'] = options.homepath
+    if options.serverUrl is None:
+        params['serverUrl'] = 'https://app.insightfinder.com'
     else:
-        parameters['serverUrl'] = options.serverUrl
+        params['serverUrl'] = options.serverUrl
     if options.chunkSize is None:
-        parameters['chunkSize'] = 50
+        params['chunkSize'] = 50
     else:
-        parameters['chunkSize'] = int(options.chunkSize)
-    return parameters
+        params['chunkSize'] = int(options.chunkSize)
+    return params
 
 
-def getAgentConfigVars():
-    configVars = {}
+def get_agent_config_vars():
```

Comment:
I think we do not use .agent.bashrc anymore. Instead, we use the config.ini in the agent code folder